docs/cspell.json

```diff
@@ -47,6 +47,7 @@
     "Dfumu",
     "Distroless",
     "Divio's",
+    "Doubleclick",
```

Comment:
Is this actually a word? Isn't it `double-click`?

Reply:
I thought double-click looked a little old-fashioned. But it seems like "double click" and "double-click" are still preferred.